articles/mfa/guides/guardian/configure-guardian.md

```diff
@@ -0,0 +1,39 @@
+---
+description: Learn how to configure Guardian Multi-factor Authentication.
```

Comment:
This page is not about configuring Guardian, it's about using a custom application that's built with the Guardian SDK, which is described here https://auth0.com/docs/multifactor-authentication/developer/libraries/android or https://auth0.com/docs/multifactor-authentication/developer/libraries/ios